filament/src/details/Camera.cpp

```diff
@@ -142,7 +152,7 @@ void UTILS_NOINLINE FCamera::setProjection(Camera::Projection projection,
     FCamera::setCustomProjection(p, c, near, far);
 }
 
-math::mat4 FCamera::getProjectionMatrix() const noexcept {
+math::mat4 FCamera::getProjectionMatrix(int eye) const noexcept {
```

Comment:
`eye` should definitely be unsigned. Use `uint8_t` or `size_t`. Also `assert_invariant` that it's 0 or 1.